doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_sdfw_sdrfw_update.rst

```diff
@@ -0,0 +1,50 @@
+.. _ug_nrf54h20_suit_sdfw_sdrfw_update:
+
+Secure Domain Firmware Update
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Both the Secure Domain Firmware (SDFW) and the recovery image for the Secure Domain Firmware are launched on a core responsible for establishing the Root of Trust as a Secure Domain.
+Due to security requirements, the update process for these binaries differs significantly from regular updates in local domains.
+
+To ensure that trusted execution can be established on the device, a valid recovery image for the Secure Domain Firmware must be present on the device before performing a Secure Domain Firmware update.
+An update is initiated only if the update candidate is verified by the Secure Domain ROM using a key that matches the requested update type.
```

Comment:
From end-user perspective: it would be great to mention that the bundle that the user programs already contains both SDFW and SDFW Recovery images, so there it no need to look for the magic "Recovery" binary.

Reply:
Added a note.